go/dpagg/mean_test.go

```diff
@@ -738,3 +738,143 @@ func TestBMFloat64Serialization(t *testing.T) {
 		}
 	}
 }
+
+func TestMeanComputeConfidenceIntervalForExplicitAlphaSplit(t *testing.T) {
+	noNoise := noNoise{} // To skip initial argument checking.
+	for _, tc := range []struct {
+		meanOpt      *BoundedMeanFloat64Options
+		sumConfInt   noise.ConfidenceInterval
+		countConfInt noise.ConfidenceInterval
+		want         noise.ConfidenceInterval // Confidence Interval after Post-processing.
+	}{
+		{ // Positive lower and upper bounds for sum.
+			meanOpt:      &BoundedMeanFloat64Options{Lower: 0.2, Upper: 15.0, Noise: noNoise, MaxContributionsPerPartition: 1},
```

Comment:
re-format this options like below

	{"default options", &BoundedMeanFloat64Options{
			Epsilon:                      ln3,
			Lower:                        0,
			Upper:                        1,
			Delta:                        0,
			MaxContributionsPerPartition: 1,
		}},

Reply:
in other tests below as well